book/content/continuous_integration/continuous_integration.md

```diff
@@ -90,7 +90,7 @@ In this chapter this entire process is referred to as continuous integration for
 <a name="What_options_are_available"></a>
 ### What are the options for CI service providers?
 
-There are many CI service providers, such as Travis, Circle CI, GitHub Actions, GitLab CI. Each of these services has its own advantages and disadvantages. In this section we provide a brief overview with links to examples to help you select the most suitable one for you.
+There are many CI service providers, such as GitHub Actions and GitLab CI. Each of these services has its own advantages and disadvantages. In this section we provide a brief overview with links to examples to help you select the most suitable one for you.
```

Comment:
```suggestion
There are many CI service providers, such as GitHub Actions and Travis CI. Each of these services has its own advantages and disadvantages. In this section we provide a brief overview with links to examples to help you select the most suitable one for you.
```

Sorry this is super picky:sweat_smile: but its nice to have two CI providers that aren't both hosted version control platforms.